apps/charterafrica/src/lib/data/common/index.js

```diff
@@ -70,6 +70,141 @@ async function processPageExplainers(page, api, context) {
+        return {
+          color,
+          label,
+          icon: { alt: icon.alt, src: icon.url },
+          href,
+        };
+      });
+
+      return {
+        background: {
+          blendMode: background.blendMode.join(","),
+          color: background.color,
+          src: background.image.url,
+        },
+        links: formattedLinks,
+        ...other,
+      };
+    });
+
+    hero.slides = heroSlides || null;
+    blocks[heroIndex] = hero;
+  }
+};
+const processHelpDesk = async (page, api, context) => {
+  const { blocks } = page;
+  const { locale, defaultLocale } = context;
+  const helpdesk = await api.findGlobal("helpdesk", {
+    locale,
+    fallbackLocale: defaultLocale,
+  });
+
+  if (helpdesk) {
+    helpdesk.slug = "helpdesk";
+    const { alt: imageAlt, url: imageSrc } = helpdesk.image || {
+      alt: null,
+      url: null,
+    };
+    helpdesk.image = { alt: imageAlt, src: imageSrc };
+    const { href: linkHref, label: linkLabel } = helpdesk.link;
+    helpdesk.link = { href: linkHref ?? null, label: linkLabel ?? null };
+    blocks.push(helpdesk);
+  }
+};
+
+const processSpotlight = (page, api, context) => {
+  const { blocks } = page;
+  const { locale } = context;
+  const spotlightIndex = blocks.findIndex(
+    (block) => block.slug === "spotlight"
+  );
+  if (spotlightIndex > -1) {
+    const spotlight = blocks[spotlightIndex];
+
+    const spotlightItems = spotlight?.items?.map((item) => {
+      const { item: itemData, ...rest } = item;
+      return {
+        ...rest,
+        item: {
+          ...itemData,
+          image: {
+            src: itemData.image.url,
+            alt: itemData.image.alt,
+          },
+          date: new Date(itemData.date).toLocaleDateString(locale, {
+            year: "numeric",
+            month: "short",
+            day: "numeric",
+          }),
+          link: {
+            href: itemData?.link.href || "#", // TODO: handle reference links
+          },
+        },
+      };
+    });
+
+    spotlight.items = spotlightItems || null;
+    blocks[spotlightIndex] = spotlight;
+  }
+};
+
+const processMooc = (page) => {
+  const { blocks } = page;
+  const moocIndex = blocks.findIndex((block) => block.slug === "mooc");
+  if (moocIndex > -1) {
+    const mooc = blocks[moocIndex];
+    if (mooc?.image?.url) {
+      mooc.image = {
+        alt: mooc.image.alt,
+        src: mooc.image.url,
+      };
+    }
+    blocks[moocIndex] = mooc;
+  }
+};
+
+async function processPageIndex(page, api, context) {
+  const { blocks } = page;
+
+  blocks.push({
+    slug: "switch",
+    startLabel: "People",
+    endLabel: "Organisations",
+  });
+  processSpotlight(page, api, context);
+  processHero(page);
+  processMooc(page);
+  processHelpDesk(page, api, context);
+  const orders = [
+    "switch",
+    "hero",
+    "spotlight",
+    "ecosystem",
+    "our-resources",
+    "mooc",
+    "helpdesk",
+    "our-partners",
+  ];
```

Comment:
Don't think we need to reorder anymore... blocks should come in the required order:

<img width="878" alt="Screenshot 2023-03-28 at 15 23 19" src="https://user-images.githubusercontent.com/1779590/228235039-d792e0ac-5a3a-4a90-a62c-8dfe77f2cb01.png">